cluster/src/main/java/org/apache/iotdb/cluster/server/member/MetaGroupMember.java

```diff
@@ -460,6 +481,57 @@ private void threadTaskInit() {
         CLEAN_HARDLINK_INTERVAL_SEC,
         CLEAN_HARDLINK_INTERVAL_SEC,
         TimeUnit.SECONDS);
+    dataClientRefresher.scheduleAtFixedRate(
+        this::refreshClientOnce, REFRESH_CLIENT_SEC, REFRESH_CLIENT_SEC, TimeUnit.SECONDS);
+  }
+
+  private void refreshClientOnce() {
+    for (Node receiver : allNodes) {
+      if (!receiver.equals(thisNode)) {
+        if (ClusterDescriptor.getInstance().getConfig().isUseAsyncServer()) {
+          refreshClientOnceAsync(receiver);
+        } else {
+          refreshClientOnceSync(receiver);
+        }
+      }
+    }
+  }
```

Comment:
Good job.
But I have some issues,  I think the `refreshClientOnce` can not **refresh** all clients in the client pool, because for one node receiver, there will be some clients, so this method can just fresh one client for one receiver node? for example, if the receiver node1 has 5 clients in the client pool, this method can only refresh one client, the other 4 clients can not be refreshed.

Reply:
>this method can only refresh one client, the other 4 clients can not be refreshed.
No. Normal `getClient` will get a client from deque's tail and then push back to the tail, so the client in deque's tail is most likely to be used a lot but the client in deque's head is mostly likely to be not used for a long time, so `refreshClientOnce` will get a client from deque's head and then push back to deque's tail.

Reply:
It looks like I might have misunderstood `pop` and `poll` in deque, I will check this :)

Reply:
You are right. I have checked the source code in `ArrayDeque`. I should use `pollLast` rather than `poll`. I'll go back to the oven and make myself again